docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +107,261 @@ To manage file storages:
    tizen.filesystem.removeStorageStateChangeListener(watchID);
    ```
 
-## Creating and Deleting Files and Directories
```

Comment:
To stop receiving the notifications, use the `removeStorageStateChangeListener()` method:

Please rephrase to:

To stop receiving the notifications, use the `removeStorageStateChangeListener()` method.

(Added a period instead of ":" to maintain consistency)

Reply:
Done.